### PR TITLE
Fix null ref exception in rigid body update

### PR DESCRIPTION
**Pull request by tombuMS**
Fix for issue #[401](https://github.com/microsoft/mixed-reality-extension-sdk/issues/401).

### CODE REVIEW

File: MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs [hunk 1]
```diff
@@ -23,10 +23,10 @@ internal class RigidBody : IRigidBody
         private Queue<Action<Rigidbody>> _updateActions = new Queue<Action<Rigidbody>>();
 
         /// <inheritdoc />
-        public MWVector3 Velocity { get; set; }
+        public MWVector3 Velocity { get; set; } = new MWVector3();
```
Comment (eanders-MS) [inline MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs]: Can `set;` be removed? Do you need to have the property setter?

Comment (tombuMS) [reply MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs]: Yes.  We set the velocity from the patching utils method.

Comment (eanders-MS) [reply MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs]: Is that in a different file? I only see Velocity.FromUnityVector3 in the Update method.

---
In reply to: [307493966](https://github.com/microsoft/mixed-reality-extension-unity/pull/95#discussion_r307493966) [](ancestors = 307493966)

Comment (eanders-MS) [reply MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs]: BTW, not a blocker, just was wondering. PR approved.

---
In reply to: [307547602](https://github.com/microsoft/mixed-reality-extension-unity/pull/95#discussion_r307547602) [](ancestors = 307547602,307493966)

Comment (tombuMS) [reply MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs]: Yes.  It is in the PatchingUtils.cs file for generating the rigid body patches.

**Pull request merged**